static/src/javascripts/projects/common/modules/onward/inject-container.js

```diff
@@ -20,10 +22,12 @@ define([
             if (resp.html) {
                 fastdom.write(function () {
                     var $el = $(containerSelector);
-                    $el.before(resp.html);
-                    $el.css({
-                        display: 'none'
-                    });
+                    $el.after(resp.html);
+                    if (!(config.page && config.page.hasStoryPackage)) {
```

Comment:
so we used to put the new stuff before the fallback, then hide the fallback. 
now we put the new stuff after, and keep the fallback if there's a story package